docs/sources/reference/components/prometheus/prometheus.operator.servicemonitors.md

```diff
@@ -32,10 +32,11 @@ prometheus.operator.servicemonitors "LABEL" {
 
 The following arguments are supported:
 
-Name | Type | Description | Default | Required
----- | ---- | ----------- | ------- | --------
-`forward_to` | `list(MetricsReceiver)` | List of receivers to send scraped metrics to. | | yes
-`namespaces` | `list(string)` | List of namespaces to search for ServiceMonitor resources. If not specified, all namespaces will be searched. || no
+Name | Type | Description                                                                                                                | Default   | Required
+---- | ---- |----------------------------------------------------------------------------------------------------------------------------|-----------| --------
+`forward_to` | `list(MetricsReceiver)` | List of receivers to send scraped metrics to.                                                                              |           | yes
+`namespaces` | `list(string)` | List of namespaces to search for ServiceMonitor resources. If not specified, all namespaces will be searched.              |           | no
+`kubernetes_role` | `string` | The Kubernetes role used for discovery. Supports `endpoints` or `endpointslice` If not specified, the `endpoints` role is used. | endpoints | no
```

Comment:
```suggestion
`kubernetes_role` | `string` | The Kubernetes role used for discovery. Supports `endpoints` or `endpointslice`. | `endpoints` | no
```

Reply:
Ah good catch... the default is implied and we don't need to state it in the description.